srcpkgs/bees/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'bees'
+pkgname=bees
+version=0.7
+revision=1
+build_style=gnu-makefile
+hostmakedepends="git discount"
```

Comment:
Why does this use git to do the build?

Reply:
https://github.com/Zygo/bees/blob/7933ccb660de3f4b5cd8d2ac2af00d4d4e6acdf3/Makefile#L10 <- this here, but looking at it again, this doesn't look all that required because of the `|| echo UNKNOWN` part

Reply:
actually, we should do something like [this](https://github.com/archlinux/svntogit-community/blob/packages/bees/trunk/PKGBUILD#L25) as well